From 44a812ea51223d82f21a098a2d45fcc5c329ce7a Mon Sep 17 00:00:00 2001
From: Tobias Stoeckmann <tobias@stoeckmann.org>
Date: Tue, 12 Mar 2019 11:46:24 +0100
Subject: [PATCH] Fix meson.build for meson 0.50.0.

Since meson 0.50.0 it is not possible anymore to specify an
absolute directory for subdir. To keep current functionality,
use install_dir instead.

atspi/meson.build:60:0: ERROR: Subdir keyword must not be an absolute path.

Signed-off-by: Tobias Stoeckmann <tobias@stoeckmann.org>
---
 atspi/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- atspi/meson.build.orig	2019-04-11 17:26:18 UTC
+++ atspi/meson.build
@@ -57,7 +57,7 @@ atspi_headers = [
 
 atspi_includedir = join_paths(get_option('prefix'), get_option('includedir'), 'at-spi-2.0', 'atspi')
 
-install_headers(atspi_headers, subdir: atspi_includedir)
+install_headers(atspi_headers, install_dir: atspi_includedir)
 
 atspi_enums = gnome.mkenums('atspi-enum-types',
                             sources: [ 'atspi-constants.h', 'atspi-types.h' ],
